packages/ruleset/src/functions/property-case-collision.js

```diff
@@ -0,0 +1,32 @@
+const { validateSubschemas } = require('../utils');
+
+module.exports = function(schema, _opts, { path }) {
+  return validateSubschemas(schema, path, propertyCaseCollision);
+};
+
+const errorMsg = 'Property names should not be identical';
```

Comment:
I think a slightly more descriptive message would be helpful, since the property names are _technically_ not identical